scss/_skeletons.scss

```diff
@@ -0,0 +1,61 @@
+// .skeleton-line      - the skeleton container
+// .skeleton-dash      - the "text" replacement
+
+.skeleton-line {
+  display: flex;
+  align-items: center;
+  width: 100%;
+
+  // We use a &nbsp; to preserve
+  // the font size of the parent
+  &::before {
+    width: 0;
+    content: "\00A0";
+  }
+
+  > .skeleton-dash {
+    width: 100%;
+    height: 1em;
+    background-color: $gray-400;
+
+    + .skeleton-dash {
+      margin-left: 4px;
+    }
+  }
+
+  // Sizing
+  &.skeleton-xs {
```

Comment:
Because we use a base-modifier pairing in our classes, you don't ned to chain these selectors. You can un-nest and omit the `&`.

Reply:
fixed